File: src/main/java/com/dec/decisland/item/custom/RedGem.java
package com.dec.decisland.item.custom;

import com.dec.decisland.attachment.ModAttachments;
import net.neoforged.neoforge.attachment.AttachmentType;

public class RedGem extends BlueGem {
    public RedGem(Properties properties) {
        super(properties);
    }
    @Override
    public int getMinAttachmentLevel() {
        return 40;
    }
    @Override
    public int getMaxAttachmentLevel() {
        return 59;
    }
    @Override
    public float getBoostValue() {return 2.0f; }
}

File: src/main/java/com/dec/decisland/item/custom/RedGem.kt
package com.dec.decisland.item.custom

class RedGem(properties: Properties) : BlueGem(properties) {
    override fun getMinAttachmentLevel(): Int = 40

    override fun getMaxAttachmentLevel(): Int = 59

    override fun getBoostValue(): Float = 2.0f
}
